packages/mui-material/src/Backdrop/Backdrop.js

```diff
@@ -174,6 +188,7 @@ Backdrop.propTypes /* remove-proptypes */ = {
    * The component used for the transition.
    * [Follow this guide](/material-ui/transitions/#transitioncomponent-prop) to learn more about the requirements for this component.
    * @default Fade
+   * @depricated Use 'slot.transition' instead. This prop will be removed in v7.
```

Comment:
typo: `@deprecated`, but the proptypes shouldn't be updated manually. Run `pnpm proptypes` to update them.